lib/api/v3/string_objects/string_object_representer.rb

```diff
@@ -34,6 +34,11 @@ module API
   module V3
     module StringObjects
       class StringObjectRepresenter < ::API::Decorators::Single
+        # accepting current_user as argument to match common interface, we do not need it though
+        def initialize(string, current_user: nil)
+          super(string, current_user: nil)
```

Comment:
As discussed the comment above the method already explains my intention.

Since this is a named parameter I also have no opportunity to rename it to `_current_user`